web/impact/impact/tests/test_graphql.py

```diff
@@ -71,7 +82,56 @@ def test_query_with_expert(self):
                             'user': {
                                 'firstName': user.first_name,
                             },
-                            'bio': user.expertprofile.bio,
+                            'bio': profile.bio,
+                            'imageUrl': (profile.image and
+                                         profile.image.url or ''),
+                            'availableOfficeHours': False,
+                            'officeHoursUrl': None,
+                            'programInterests': []
+                        }
+                    }
+                }
+            )
+
+    def test_office_url_field_returns_correct_value(self):
+        with self.login(email=self.basic_user().email):
```

Comment:
For easier reading, I'd rather see the `with` block contain the lines where login is relevant (i.e. the `self.client.post` call in line 125 below) rather than containing all the setup code as well.